test/eventrouter.sh

```diff
@@ -6,7 +6,7 @@ source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
 source "${OS_O_A_L_DIR}/hack/testing/util.sh"
 os::test::junit::declare_suite_start "test/eventrouter"
 
-FLUENTD_WAIT_TIME=${FLUENTD_WAIT_TIME:-$(( 3 * minute ))}
+FLUENTD_WAIT_TIME=${FLUENTD_WAIT_TIME:-$(( 5 * minute ))}
```

Comment:
do you have evidence that part of the problem is that it takes fluentd too long to process eventrouter records?

Reply:
I could reduce the time and rerun the test.  Do you want me to make it shorter in https://github.com/openshift/origin-aggregated-logging/pull/1437, as well?